packages/jellyfish-api-core/__tests__/category/vault/placeAuctionBid.test.ts

```diff
@@ -464,13 +478,13 @@ describe('placeAuctionBid success', () => {
           batches: [
             {
               index: 0,
-              collaterals: ['5004.44437154@DFI', '0.49955556@BTC'], // 5004.44437154 + 4995.5556 = 9999.99997154
-              loan: '499.56012216@TSLA' // 499.56012216 * 15 = 7493.4018324
+              collaterals: ['5004.44449284@DFI', '0.49955555@BTC'],
```

Comment:
@surangap  All these values should remain the same.

Reply:
Thought this was reverted back. should be fine now.